RELEASE/scripts/autoscend/quests/optional.ash

```diff
@@ -690,21 +690,30 @@ boolean LX_pirateQuest() {
 	return false;
 }
 
-string[class] legendaryEpicWeapons;
-legendaryEpicWeapons[$class[Seal Clubber]] = "Hammer of Smiting";
-legendaryEpicWeapons[$class[Turtle Tamer]] = "Chelonian Morningstar";
-legendaryEpicWeapons[$class[Pastamancer]] = "Greek Pasta Spoon of Peril";
-legendaryEpicWeapons[$class[Sauceror]] = "17-alarm Saucepan";
-legendaryEpicWeapons[$class[Disco Bandit]] = "Shagadelic Disco Banjo";
-legendaryEpicWeapons[$class[Accordion Thief]] = "Squeezebox of the Ages";
-// usage: item legendaryEpicWeapon = legendaryEpicWeapons[my_class()].to_item();
+item[class] legendaryEpicWeapons;
```

Comment:
does my_class() capitalize the way you did and would capitalization matter since they are converted to $class

Reply:
```
> ash my_class()

Returned: Disco     Bandit
```

also it doesn't matter.